### ISSUE TITLE
Error on Save window layout

### ISSUE THREAD

**Issue by cg-mayrhofer** [spyder-ide/spyder#15368]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [ ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
Can't save window layout

### What steps reproduce the problem?

1. Created an own window layout. Mainly editor on the right, see screenshot
![spl](https://user-images.githubusercontent.com/77964479/115512781-a0abe480-a282-11eb-9cdb-81a856c739b2.jpg)

2. Then trying to save the layout using *view-->windows layout-->save current*

### Paste Traceback/Error Below (if applicable)

```python-traceback

Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\layout\container.py", line 131, in <lambda>
    triggered=lambda: self.show_save_layout(),
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\layout\container.py", line 307, in show_save_layout
    self.save_current_window_settings('layout_{}/'.format(index),
AttributeError: 'LayoutContainer' object has no attribute 'save_current_window_settings'

```

## Versions

* Spyder version: 5.0.0
* Python version: 3.7.7
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: win 10 64

### Dependencies

```
PASTE DEPENDENCIES HERE
```

**Comment by ccordoba12** [comment]
Hey @cg-mayrhofer, thanks for reporting. This is already fixed in our 5.0.1 version, released last Friday and that will be part of Anaconda soon.

**Comment by cg-mayrhofer** [comment]
Great thanks! Indeed I am using Anaconda...
